util/time_test.go

```diff
@@ -0,0 +1,87 @@
+package util
+
+import (
+	"fmt"
+	"testing"
+)
+
+//Time zone should be considered for time related operations
+func TestFormatTimeMillis(t *testing.T) {
+	type args struct {
+		ts uint64
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			"test1",
+			args{1582421778887},
+			"2020-02-23 01:36:18", //"2020-02-23 09:36:18",
```

Comment:
These test cases will fail in non-UTC-zone machines. We may need to subtract the time zone offset. For example:

```go
_, offset := time.Now().Zone()
offsetMs := uint64(offset * 1000)
```

Reply:
👌

Reply:
already fix